src/dotnet/commands/dotnet-add/dotnet-add-p2p/Program.cs

```diff
@@ -95,7 +95,8 @@ internal static CommandLineApplication CreateApplication(CommandLineApplication
                         }
                     }
 
-                    msbuildProj.ConvertPathsToRelative(ref references);
+                    references = references.Select((r) =>
+                        PathUtility.GetRelativePath(msbuildProj.ProjectDirectory, Path.GetFullPath(r))).ToList();
```

Comment:
This is effectively changing the meaning of this variable from (implictly) something like `absolutePathReferences` to `relativePathReferences`. Two variables with explicit names would make the code clearer.